python/nano/tutorial/notebook/training/pytorch/convert_pytorch_training_torchnano.ipynb

```diff
@@ -0,0 +1,284 @@
+    "4. Call `TorchNano`'s `setup` method to set up model, optimizer(s), and dataloader(s) for accelerated training\n",
+    "5. Replace `loss.backward()` with `self.backward(loss)`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Step 1. import TorchNano\n",
+    "from bigdl.nano.pytorch import TorchNano\n",
+    "\n",
+    "# Step 2. subclass TorchNano and override its train method\n",
+    "class MyNano(TorchNano):\n",
+    "    def train(self):\n",
+    "        # Step 3. Move the code for your custom training loops inside the train method\n",
+    "        model = MyPytorchModule()\n",
+    "        optimizer = torch.optim.SGD(model.parameters(), lr=0.01, momentum=0.9, weight_decay=5e-4)\n",
+    "        loss_fuc = torch.nn.CrossEntropyLoss()\n",
+    "        train_loader, val_loader = create_dataloaders()\n",
+    "\n",
+    "        # Step 4. call setup method to set up model, optimizer(s), \n",
+    "        #         and dataloader(s) for accelerated training\n",
+    "        model, optimizer, (train_loader, val_loader) = self.setup(model, optimizer,\n",
+    "                                                                  train_loader, val_loader)\n",
+    "        model.train()\n",
+    "        num_epochs = 5\n",
+    "\n",
+    "        for epoch in range(num_epochs):\n",
+    "            train_loss, num = 0, 0\n",
+    "            for data, target in train_loader:\n",
+    "                output = model(data)\n",
+    "                loss = loss_fuc(output, target)\n",
+    "\n",
+    "                optimizer.zero_grad()\n",
+    "                # Step 5. Replace loss.backward() with self.backward(loss)\n",
+    "                self.backward(loss)\n",
+    "                optimizer.step()\n",
+    "\n",
+    "                train_loss += loss.sum()\n",
+    "                num += 1\n",
+    "            print(f'Train Epoch: {epoch}, avg_loss: {train_loss / num}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You could then do the training by instantiating `MyNano` and calling its `train` method:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "MyNano().train()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "> 📝 **Note**\n",
+    ">\n",
+    "> Due to the optimized environment variables set by `source bigdl-nano-init`, you could already experience some training acceleration after converting your PyTorch code to use `TorchNano`."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "> 📚 **Related Readings**\n",
+    "> \n",
+    "> - [How to install BigDL-Nano](https://bigdl.readthedocs.io/en/latest/doc/Nano/Overview/nano.html#install)"
```

Comment:
@Siritao After adding more methods related notebooks (ipex, bf16, etc.), you could link them in this notebook here like the follows:
![image](https://user-images.githubusercontent.com/54161268/206953592-246e3349-be8f-4d3a-a503-014debf43a1b.png)